### PR TITLE
Css canvas medical

### PR DESCRIPTION
**Pull request by hnguy96**
Screen Shot 2021-03-16 at 9 52 25 AM
Screen Shot 2021-03-16 at 9 52 17 AM
Screen Shot 2021-03-16 at 9 51 54 AM
Screen Shot 2021-03-16 at 9 52 33 AM
Screen Shot 2021-03-16 at 9 56 15 AM
Screen Shot 2021-03-16 at 9 57 00 AM
Screen Shot 2021-03-16 at 9 52 54 AM
Screen Shot 2021-03-16 at 9 57 43 AM
Screen Shot 2021-03-16 at 9 58 00 AM
Screen Shot 2021-03-16 at 9 58 06 AM

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Hien! The text in the header and footer "Canvas" is actually an image we've provided in the `images` folder, not text 😁 . Please use those images and submit again with a new screenshot, and I will take another look!

**Comment by hnguy96** [comment]
Screen Shot 2021-03-16 at 12 59 58 PM
Screen Shot 2021-03-16 at 12 59 36 PM

File: css-canvas-medical/styles.css [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+}
+
+.locations {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.location-heading {
+  max-width: 1300px;
+  max-width: 600px;
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:140]: Whoops, duplicated property here! The one being used is the bottom one, so you can remove the top one.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Very nice work!! One more minor change:

**Comment by hnguy96** [comment]
Screen Shot 2021-03-16 at 7 38 39 PM

Removed duplicated property in the `.location-heading` class because I realized it had no reason being there.

**Review by codingcodymiller — APPROVED** [review]
Nice work!!! 🥳 🥳 🥳

**Pull request merged**